fdbserver/RatekeeperInterface.h

```diff
@@ -75,6 +75,34 @@ struct ClientTagThrottleLimits {
 	}
 };
 
+struct TransactionCommitCostEstimation {
+	int64_t numWrite = 0;
+	int64_t numAtomicWrite = 0;
+	int64_t numClear = 0;
+	unsigned long bytesWrite = 0;
+	unsigned long bytesAtomicWrite = 0;
+	Optional<unsigned long> numClearShards;
+	Optional<unsigned long> bytesClearEst;
```

Comment:
For these fields I think it would be simpler to not use `Optional`, and the fields can just default to 0.